PURL-TYPES.rst

```diff
@@ -59,6 +59,29 @@ apk
       pkg:apk/alpine/curl@7.83.0-r0?arch=x86
       pkg:apk/alpine/apk@2.12.9-r3?arch=x86
 
+bazel
+-----
+``bazel`` for Bazel modules:
+
+- The default repository ("registry") is ``https://bcr.bazel.build``, the
+  Bazel Central Registry (BCR).
+- The ``name`` is the module name. It must be lowercased.
+- The ``version`` is the module version in `Bazel's relaxed semver format
+  <https://bazel.build/external/module#version_format>`_.
+- The optional ``repository_url`` can be used to specify the URL of an
+  alternative registry, with any trailing forward slashes removed.
+- The optional ``subpath`` can name a particular Bazel target in the module via
+  a label with the leading double slash (``//``) removed and canonicalized by
+  omitting the target name if it is equal to the name of the containing package.
```

Comment:
Hmm. The [spec](https://github.com/package-url/purl-spec/blob/master/PURL-SPECIFICATION.rst#package-url-specification-v10x) describes the `subpath` as

> extra subpath within a package, relative to the package root.

How is what you describe "relative to the package root"? Would a custom qualifier maybe make more sense to store the Bazel target?

Reply:
I simplified this part of the spec in a new commit so that it's always a regular file path, corresponding to a package (not target or file) in the Bazel module. This is analogous to the usage of `subpath` for `golang`.

Reply:
Looks good to me now, but I'd like others to share their opinion as well.